addons/nfsserverprovisioner/2.3.x/nfsserverprovisioner-1.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  annotations:
+    appversion.kubeaddons.mesosphere.io/nfs-server-provisioner: 2.3.0
+    catalog.kubeaddons.mesosphere.io/addon-revision: 0.6.0-1
```

This should reference the appversion, not the chart version (as some addons are helm, some are kudo we use the app version).
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: 2.3.0-1
```